packages/theme/src/theme/_animation.scss

```diff
@@ -2,6 +2,29 @@
 	animation: $animation-name 1.5s ease infinite;
 }
 
+@mixin tc-animation($animation-name, $duration, $function, $iteration) {
+	animation: $animation-name $duration $function $iteration;
```

Comment:
request change: I don't see this mixin as useful. the content is shorten than a call to the mixin.
I think it's easier to directly use animation no ?